UI/nix-update/crypto-helpers-mac.mm

```diff
@@ -0,0 +1,50 @@
+#include "crypto-helpers.hpp"
+
+#import <Foundation/Foundation.h>
+#import <Security/Security.h>
+#import <Security/SecKey.h>
+
+bool VerifySignature(const uint8_t *buf, const size_t len, const uint8_t *sig,
+		     const size_t sigLen)
+{
+	NSString *pubKeyStr =
```

Comment:
Have you looked at https://developer.apple.com/documentation/security/1395728-secitemimport?language=objc which supports `kSecFormatPEMSequence`?

Reply:
I have not, I probably disregarded it while reading documentation because of the "Import" part, but it looks like you can use it to load a key without actually importing it. I'll give it a shot tomorrow.